src/caja-navigation-window-menus.c

```diff
@@ -689,11 +709,13 @@ action_show_hide_search_callback (GtkAction *action,
 
     window = CAJA_NAVIGATION_WINDOW (user_data);
 
+    G_GNUC_BEGIN_IGNORE_DEPRECATIONS;
     if (gtk_toggle_action_get_active (GTK_TOGGLE_ACTION (action)))
     {
         caja_navigation_window_show_search (window);
     }
     else
+    G_GNUC_END_IGNORE_DEPRECATIONS;
     {
```

Comment:
```
    else
    G_GNUC_END_IGNORE_DEPRECATIONS;
    {
```

it seems to be wrong

Reply:
I know, i am thinking about to make at better as here (already posted).
https://github.com/mate-desktop/caja/pull/1242#issuecomment-506695298

Reply:
last commit fixes caja-search.
I hope ccpcheck tool is happy.